### ISSUE TITLE
Fix editor crash when it tries to load AccessibilityProviders

### ISSUE THREAD

**Issue by vlaaad** [defold/defold#8001]
Context: https://forum.defold.com/t/defold-immediately-crashes-upon-launching/74053/9

There is a file `.accessibility.properties` in the user's home directory. It makes the JVM try to load a class `com.sun.java.accessibility.AccessBridge`, which does not exist on the classpath, which makes the editor crash on start.

**Task (REQUIRED):**
Figure out a way so that the editor does not crash when such a file is present.

**Expected outcome (REQUIRED):**
The editor does not crash on startup.

**Comment by nathnolt** [comment]
Also nothing is written into the log file about Defold not starting up due to this file being present into the home directory.

**Comment by vlaaad** [comment]
@nathnolt have you had this problem?

**Comment by nathnolt** [comment]
Yes, I did have this problem, and looking around on the internet for "Defold not starting" or "defold not working" will give you results of more people having this same issue. So this is certainly an important thing to fix in my opinion.
